model/googleAuth.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+
+const googleAuthSchema = new mongoose.Schema({
+    google_id: {
+        type: Number,
+        required: true
+    },
+    display_name: {
+        type: String,
+        required: true
+    },
+    first_name: {
```

Comment:
# Google Oauth Returning without  First name and Last Name

There have been instances in which the user signup Google Service without providing their name or other details [Read This](https://stackoverflow.com/questions/58316184/get-empty-first-name-and-last-name-for-some-users-from-google-oauth) 

Did You Make a **Fail Use Case** For This?

Reply:
This Field Are Kept As Required

Reply:
Oh, I never it's possible for Google service not to return some of those fields. Okay I'll make appropriate changes to that.